doc/syntax.markdown

```diff
@@ -23,42 +24,71 @@ The general structure looks like:
 	//
 	// A more detailed multi-line description.
 	//
+	// Query: $ref: QueryObj
 	// Request body: $ref: RequestObj
 	// Response 200: $ref: AnObject
 	// Response 400: $ref: ErrorObject
 
-
 Opening line, tagline, and description
 --------------------------------------
+Every Kommentaar comment block must start with a description of the path as:
+
+	VERB /path [tag]
+
+- The `VERB` is a valid HTTP verb; it *must* be in upper-case.
+- The `/path` is a HTTP path. Path parameters can be added as `{..}`.
+- An optional tag can be added for categorisation.
+
+The second line is used as a "tagline". This can only be a single line and
+*must* immediately follow the opening line with no extra newlines. This line is
+optional but highly recommended to use.
+
+The tagline can be of any length, but it is highly recommended that it is kept
+short and concise.
 
-    POST /bike bikes
-    Create a new bike.
+After a single blank line any further text will be treated as the endpoint's
+description. This is free-form text and may be omitted (especially in cases
+where it just repeats the tagline it's not useful to add).
 
-    This will create a new bike. It's important to remember that newly created
-    bikes are *not* automatically fit with a steering wheel or seat, as the
-    customer will have to choose one later on.
+Full example:
+
+    POST /bike/{shedID} bikes
+    Create a new bike and store it in the given shed.
+
+	It's important to remember that newly created bikes are *not* automatically
+	fit with a steering wheel or seat, as the customer will have to choose one
```

Comment:
What kind of crazy bikes are you riding?! All mine have handle bars :biking_man: